sklearn/utils/deprecation.py

```diff
@@ -114,3 +114,22 @@ def _is_deprecated(func):
         [c.cell_contents for c in closures if isinstance(c.cell_contents, str)]
     )
     return is_deprecated
+
+
+# TODO: remove in 1.7
+def _deprecate_Xt_in_inverse_transform(X, Xt):
+    """Helper to deprecate the `Xt` argument in favor of `X` in inverse_transform."""
+    if X is not None and Xt is not None:
+        raise TypeError("Cannot use both X and Xt. Use X only.")
+
+    if X is None and Xt is None:
+        raise TypeError("Missing required positional argument: X.")
+
+    if Xt is not None:
+        warnings.warn(
+            "Xt was renamed X in version 1.5 and will be removed in 1.7.",
```

Comment:
```suggestion
            "'Xt' was renamed 'X' in version 1.5 and will be removed in 1.7.",
```

Reply:
we agreed to not put them between quotes, see https://github.com/scikit-learn/scikit-learn/pull/28756#discussion_r1582929071